vnext/Microsoft.ReactNative/Base/CoreUIManagers.cpp

```diff
@@ -61,10 +57,7 @@ void AddStandardViewManagers(
 void AddPolyesterViewManagers(
```

Comment:
nit, you can just remove this method and move the call to add the glyph view manager onto the caller of this method which there probably is only one anyway